spec/views/lectures/show.html.erb_spec.rb

```diff
@@ -43,6 +43,13 @@
       expect(rendered).to have_css("#enrollmentKey-tab")
     end
 
+    it "renders enrollment qr code if enrollment key is present" do
+      @qr_code = RQRCode::QRCode.new("http://some-random.url/that/is/not/tested")
+      render
+      assert_select "a", "Enrollment Key"
+      expect(rendered).to have_selector("div", :class => "qr-code-container")
+    end
+
     it "does not render enrollment key tab button if enrollment key is not present" do
       @lecture.update(enrollment_key: nil)
       render
```

Comment:
no bdd test for scanning in a qr code and then checking if the user is redirected and saved as a participant to that corresponding lecture?

Reply:
Sorry but at test for scanning a qrcode is in my opinion too hard and out of scope